parity-instantseal/instant-seal-config.toml

```diff
@@ -52,7 +54,7 @@ disable = true
 
 [ipfs]
 # You won't be able to hash-query blockchain data.
-disable = true
+enable = false
```

Comment:
Comment above is now incorrect.

Reply:
The comment remains true, they just changed the nature of the variable